setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-name = sophus
+name = sophus-protos
 version = 0.1.0-dev
```

Comment:
package name and version TBD -- @strasdat @stevenlovegrove @ethanrublee

Reply:
As discussed, I think sophus is a good package name. It allows us to expand easily in the future python package to include sophus python implementation, and parks the official package name.

The project description can be modified to include the current limitation of this library that it includes only the protobuf definitions.